src/utils/fromDateToString.ts

```diff
@@ -9,3 +9,29 @@ export const fromDateToString = (arg: Date) => {
       : new Date(arg).getDate()
   }`;
 };
+
+export const createDatesArray = (startDate: string, stopDate: Date, whenToRecur: string) => {
+  const dateArray = [];
+  let currentDate = new Date(startDate);
+  while (currentDate <= stopDate) {
+    switch (whenToRecur) {
```

Comment:
what happens if it doesn't _enter_ in any case

Reply:
The default value of whenToRecur is "day".